### PR TITLE
Add multi-language support to SendEmail action

### PR DESCRIPTION
**Pull request by vova07**

### CODE REVIEW

File: spec/components/schemas/Rules/RulesEmailNotification.yaml [hunk 1]
```diff
@@ -37,24 +37,39 @@ properties:
...
+            The template placeholders are allowed.
+        editor:
+          type: string
+          description: |
+            The source of the message required for the email editor.
+            Not used for sending emails.
+            Used by the editor to reproduce the message for future updates.
+    description: An array of body objects maped to the language locale indetifiers according to RFC 4646
+    example:
+      en-US:
```
Comment (slavcodev) [inline spec/components/schemas/Rules/RulesEmailNotification.yaml]: It's not clear what would happen if locale is not found? let say `en-US` is defined, but we need `en-UK`?

File: spec/components/schemas/Rules/RulesEmailNotification.yaml [hunk 2]
```diff
@@ -37,24 +37,39 @@ properties:
...
-      The message's html body.
-      Leave empty to use content from "bodyText".
-      The template placeholders are allowed.
-  editor:
-    type: string
-    description: |
-      The source of the message required for the email editor.
-      Not used for sending emails.
-      Used by the editor to reproduce the message for future updates.
+  body:
```
Comment (slavcodev) [inline spec/components/schemas/Rules/RulesEmailNotification.yaml]: What about `subject`? Should not it also be localized?

File: spec/components/schemas/Rules/RulesEmailNotification.yaml (hunk 1 above)
Comment (vova07) [reply spec/components/schemas/Rules/RulesEmailNotification.yaml]: I think in that case we'll use the `default` localization.
However I still have some open questions I wait for response related with this, like: is the `en-US` always required and the default one?!

File: spec/components/schemas/Rules/RulesEmailNotification.yaml (hunk 2 above)
Comment (vova07) [reply spec/components/schemas/Rules/RulesEmailNotification.yaml]: I think it's a good questions.

Thanks!

File: spec/components/schemas/Rules/RulesEmailNotification.yaml (hunk 1 above)
Comment (slavcodev) [reply spec/components/schemas/Rules/RulesEmailNotification.yaml]: This is my concern. What is default? Why it should be `en-US`? In theory our clients may not use `en-US` at all.

File: spec/components/schemas/Rules/RulesEmailNotification.yaml [hunk 3]
```diff
@@ -37,24 +37,39 @@ properties:
...
+            The message's html body.
+            Leave empty to use content from "text".
+            The template placeholders are allowed.
+        editor:
+          type: string
+          description: |
+            The source of the message required for the email editor.
+            Not used for sending emails.
+            Used by the editor to reproduce the message for future updates.
+    description: An array of body objects maped to the language locale indetifiers according to RFC 4646
```
Comment (adamaltman) [inline spec/components/schemas/Rules/RulesEmailNotification.yaml]: copy/paste into a spell-checker.

File: spec/components/schemas/Rules/RulesEmailNotification.yaml (hunk 1 above)
Comment (adamaltman) [reply spec/components/schemas/Rules/RulesEmailNotification.yaml]: The default customer locale is US English.  Maybe in the future we'll have the ability to have a merchant-level default customer locale.  Whatever the default, it should appear here.

**Comment by vova07** [comment]
Closing this in favor of: https://github.com/Rebilly/RebillyUserAPI/pull/342

**Pull request closed without merging**